adapters/beintoo/beintoo.go

```diff
@@ -0,0 +1,222 @@
+package beintoo
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"net/url"
+	"strconv"
+
+	"github.com/mxmCherry/openrtb"
+	"github.com/prebid/prebid-server/adapters"
+	"github.com/prebid/prebid-server/errortypes"
+	"github.com/prebid/prebid-server/openrtb_ext"
+)
+
+type BeintooAdapter struct {
+	endpoint string
+}
+
+func (a *BeintooAdapter) MakeRequests(request *openrtb.BidRequest, reqInfo *adapters.ExtraRequestInfo) ([]*adapters.RequestData, []error) {
+	var errs []error
+
+	if len(request.Imp) == 0 {
+		return nil, []error{&errortypes.BadInput{
+			Message: fmt.Sprintf("No Imps in Bid Request"),
+		}}
+	}
+
+	if errs := preprocess(request); errs != nil && len(errs) > 0 {
+		return nil, append(errs, &errortypes.BadInput{
```

Comment:
`errs` is redefined in this scope from the func scope of the error collection. Go allows for variable shadowing. Appending to `errs` is not appending to `errs` from line 21 but the `errs` from the preprocess call. Your unit tests correctly account for this behavior. I just want to verify that's what you want to happen.

Reply:
Hi @SyntaxNode for me it is ok.

Reply:
Ok.